command/smartContract.go

```diff
@@ -0,0 +1,55 @@
+package command
+
+import (
+	"github.com/rubixchain/rubixgoplatform/client"
+	"github.com/rubixchain/rubixgoplatform/core"
+)
+
+func (cmd *Command) generateSmartContractToken() {
+	smartContractTokenRequest := core.GenerateSmartContractRequest{
```

Comment:
I believe this variable is not required, below one is sufficient and value can be directly assigned